pkg/registry/core/node/strategy.go

```diff
@@ -67,7 +67,6 @@ func (nodeStrategy) AllowCreateOnUpdate() bool {
 func (nodeStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
 	node := obj.(*api.Node)
 	// Nodes allow *all* fields, including status, to be set on create.
-
 	if !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
 		node.Spec.ConfigSource = nil
```

Comment:
also set `node.Status.Config = nil` here

node creation allows setting status, which is unusual (and only allowed for backwards compatibility with early resource behavior)

Reply:
done